python/mlcroissant/mlcroissant/_src/core/ml/bounding_box.py

```diff
@@ -0,0 +1,53 @@
+"""Module to manage "bounding boxes" annotations on images."""
+
+import enum
+from typing import Any
+
+
+# TODO(marcenacp): Coco supports XYWH.
+class BoundingBoxFormat(enum.Enum):
+    """The format of the bounding box to extract from.
+
+    Note:
+    - Croissant uses XYXY as the common format.
```

Comment:
TODO?

Reply:
I chose to follow [Keras' bounding box formats](https://keras.io/api/keras_cv/bounding_box/formats). Croissant will have a standard output format (not sure yet whether it will be `XYXY` or `XYWH`).

Reply:
I ended up removing entirely this class: `mlcroissant` won't check for the format for now.